### ISSUE TITLE
About the SAN losses

### ISSUE THREAD

**Issue by sh-lee-prml** [sony/bigvsan#3]
Thanks for nice work 👍

Now, I'm trainining my TTS model by replacing BigVGAN with BigVSAN. 

First, the model with BigVSAN shows slightly better results in early steps! 🚀🚀🚀

I have attached tensorboard graph and I was wondering if maybe you could have similar results during training. 

![image](https://github.com/sony/bigvsan/assets/56749640/87440420-61ee-4bb5-ae3a-1905179559e0)

Have you tried to tune the hyper-parameter for gen loss or feature matching loss? The scales of these losses are quite different from the baseline model (BigVGAN),  

![image](https://github.com/sony/bigvsan/assets/56749640/fee5b3a9-eab3-4a3c-b76e-15b4bc0b52df)

Thanks again 😊

**Comment by TakashiShibuyaSony** [comment]
Thank you for your interest and sharing your learning curves!

That's a good question. There should be room for improvement in hyperparameter setting. We didn't conduct any hyperparameter tuning, and just compared BigVSAN and BigVGAN, giving the same hyperparameter values. We're surely interested in how largely the performance will improve after elaborate hyperparameter tuning, but we're spending our time doing other things now.

**Comment by sh-lee-prml** [comment]
Could you share your learning curves? 

I hope to have a handle on training SAN 👀

**Comment by TakashiShibuyaSony** [comment]
Here they are!
![image](https://github.com/sony/bigvsan/assets/107020494/d02ea09f-3c74-42c2-bcd4-3c5e14a77262)
![image](https://github.com/sony/bigvsan/assets/107020494/46deba49-209c-4324-8673-6747b42c76e9)

We didn't record a loss for each discriminator as you're doing. We have only information on total losses for multiple discriminators. Light blue curves are for our BigVSAN, and pink ones for our BigVGAN reproduction. I hope these are informative.
